testing/src/main/java/com/firebase/geofire/testing/TestCallback.java

```diff
@@ -1,5 +1,6 @@
 package com.firebase.geofire.testing;
 
+import static com.firebase.geofire.testing.GeoFireTestingRule.TIMEOUT_SECONDS;
```

Comment:
nit: I am not a huge fan of `import static` outside of assert/mock methods since it makes it a little bit harder to find the source.